himembind.cpp

```diff
@@ -142,19 +142,20 @@ bool is_valid_arguments(char **args, int argc)
  * @param call_name
  * @return result
  */
-bool is_valid_call_name(const char *call_name) {
-    std::string call_name_str(call_name);
-    std::string_view valid_names[] = {
-            "python",
-            "python3"
-    };
-    for (const auto &name: valid_names) {
-        if (call_name_str.size() >= name.size()) {
-            if (std::equal(std::crbegin(call_name_str), std::crbegin(call_name_str) + name.size(),
-                           std::crbegin(name))) {
-                return true;
-            }
-        }
-    }
-    return false;
+bool is_valid_call_name(const std::string call_name) {
+     constexpr const char* valid_names[] = {
+          "python",
+          "python3"
```

Comment:
Sometimes it is required to run `python` calling it with full absolute path, e.g. `mpirun -np 2 himembind /home/user/very/log/path/to/env/bin/python shor_mpi.py`. So `himembind` must correctly deal with this situation and at least allow any valid absolute path. 

**Any way I want to emphasize that instead of Python it is presumed that `himembind` should allow any command valid for `mpirun` .**  This was a main intention for `himembind` development. The restriction only to Python will make it much less usefull. *I don't see any security risk allowing any command without restrictions, because at the same time the same command can be executed by user without using `himembind`.*

Reply:
Thank you for your clarification @yzotov. In that case, I think the issue was probably flagged by some automated testing suite.
In that case, I would also vote for dropping any checks on the name of the command being run by `himembind` and any checks on its arguments.